.github/workflows/merge-prometheus.yaml

```diff
@@ -18,10 +18,6 @@ jobs:
         go.mod
         go.sum
         .golangci.yml
-      assets-cmd: |
-        make assets
-        git add web/ui/assets_vfsdata.go
```

Comment:
Instead of removing the assets generation step, you may need to change the existing to the following.

```
make assets-compress
find web/ui/static/react -type f -name '*.gz' -exec git add {} -f \;
git diff --cached --exit-code || git commit -s -m "[bot] assets: generate"
```

Reply:
Yes, thank you. I forgot about that. I added you suggestion plus another needed addition.